packages/SwingSet/src/kernel/vat-warehouse.js

```diff
@@ -589,12 +591,20 @@ export function makeVatWarehouse({
     } else if (deliveriesInSpan >= snapshotInterval) {
       // begin snapshot after 'snapshotInterval' deliveries in a span
       reason = { snapshotInterval };
+    } else if (forceThreshold > 0 && deliveriesInSpan > forceThreshold) {
+      reason = { forceThreshold };
```

Comment:
This should follow the existing non-strict `>=` comparison already used for `snapshotInitial` and `snapshotInterval`.
```suggestion
    } else if (deliveriesInSpan >= minDeliveryCount) {
      reason = { minDeliveryCount };
```